recipes/entrypoints/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp entrypoints.py $SP_DIR
```

Comment:
Probably @takluyver can say more, but I guess it is just a `.py` that we need to drop somewhere in the path.